src/opera/test/pge/dist_s1/test_dist_s1_pge.py

```diff
@@ -108,16 +166,255 @@ def test_dist_s1_pge_execution(self):
         expected_log_file = pge.logger.get_file_name()
         self.assertTrue(os.path.exists(expected_log_file))
 
-        # Lastly, check that the dummy output products were created
-        tif_files = glob.glob(join(pge.runconfig.output_product_path, "*.tif"))
-        self.assertEqual(len(tif_files), 1)
```

Comment:
Why was this removed and not just updated to test for the actual number of output tifs we expect?

Reply:
I have no idea how those lines got deleted....
Definitely didn't intend to alter that case or don't recall if/why I would